### ISSUE TITLE
[FEATURE REQUEST] Real Commvault modules

### ISSUE THREAD

**Issue by stobias123** [CommvaultEngg/ansible#8]
Please provide real ansible modules, rather than a light REST wrapper. 

A cursory glance makes it seem like all "operations" should be independent modules, each with proper params, rather than freeform json blobs.

**Comment by ddill** [comment]
I agree.  We also need enhancements to handle more types of databases (e.g. Oracle RAC)

**Comment by keilr** [comment]
An [ansible collection](https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html) to easily reuse the modules would be nice. CI-Pipeline/Testing should be considered.
